test/plato_test.js

```diff
@@ -130,5 +129,18 @@ exports['plato'] = {
       test.ok(overview.summary.total.jshint === 2, 'Should contain total jshint issues');
       test.done();
     });
+  },
+  'test file ending checks' : function(test) {
+    var files = [
+      'test/fixtures/test.es'
+    ];
+    
+    text.expect(1);
```

Comment:
@MathRobin sorry your response got cut off, instead of what? If you look at the files changed tab, GitHub says it was added I believe.